bin/reth/src/args/rpc_server_args.rs

```diff
@@ -354,7 +357,7 @@ impl RethRpcConfig for RpcServerArgs {
     }
 
     fn rpc_max_response_size_bytes(&self) -> u32 {
-        self.rpc_max_response_size.saturating_mul(1024 * 1024)
+        self.rpc_max_response_size.0.saturating_mul(1024 * 1024)
```

Comment:
can we replace this with a `fn get() {self.0}` instead?

Reply:
> can we replace this with a `fn get() {self.0}` instead?

sure, going to work on this

Reply:
> can we replace this with a `fn get() {self.0}` instead?

does this looks better?